<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoStack</class>
 <widget class="QWidget" name="InfoStack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>186</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>186</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>186</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
color: #607496;
font-family: Roboto, Helvetica;
font-size: 12px;
font-weight: 400;
}
#valueGivenNames, #valueSurname,
#valuePersonalCode, #valueCitizenship,
#valueDocument, #valueExpiryDate {
color: #003168;
font-size: 14px;
font-weight: 700;
}
#valueExpiryDate[label=&quot;error&quot;] {
padding: 2px 10px;
border-radius: 8px;
color: #AD2A45;
background: #F5EBED;
font-weight: 400;
}
#valueExpiryDate[label=&quot;good&quot;] {
padding: 2px 10px;
border-radius: 8px;
color: #1A641B;
background: #EAF8EA;
font-weight: 400;
}</string>
  </property>
  <layout class="QGridLayout" name="InfoStackLayout" columnstretch="0,1">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <property name="horizontalSpacing">
    <number>40</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="labelGivenNames">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Given names</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelSurname">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Surname</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="valueGivenNames">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">valueGivenNames</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="valueSurname">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">valueSurname</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPersonalCode">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Personal code</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelCitizenship">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Citizenship</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="valuePersonalCode">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">valuePersonalCode</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="valueCitizenship">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">valueCitizenship</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelExpiryDate">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Expiry date</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelDocument">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string>Document</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="valueExpiryDate">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">valueExpiryDate</string>
     </property>
     <property name="label" stdset="0">
      <string notr="true">good</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="valueDocument">
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="text">
      <string notr="true">valueSerialNumber</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>labelGivenNames</tabstop>
  <tabstop>valueGivenNames</tabstop>
  <tabstop>labelSurname</tabstop>
  <tabstop>valueSurname</tabstop>
  <tabstop>labelPersonalCode</tabstop>
  <tabstop>valuePersonalCode</tabstop>
  <tabstop>labelCitizenship</tabstop>
  <tabstop>valueCitizenship</tabstop>
  <tabstop>labelExpiryDate</tabstop>
  <tabstop>valueExpiryDate</tabstop>
  <tabstop>labelDocument</tabstop>
  <tabstop>valueDocument</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
